server/index.js

```diff
@@ -63,6 +69,16 @@ function start(portIncomming) {
             console.log('Asset Compute Developer Tool Server started on url ', assetComputeDevToolUrl);
             await open(assetComputeDevToolUrl);
         } else {
+
+            // read/process package.json
+            const file = '../client/package.json';
+            const pkg = fse.readJSONSync(file, { throws: false });
+
+            pkg.proxy = `http://localhost:${port}`;
+
+            // Write to package.json
+            fse.writeJSONSync(file, pkg, { spaces: '\t' });
```

Comment:
Well, that is probably the reason why indentation feels off in that file: `{ spaces: '\t' }` looks wrong. Stick with ` { spaces: 4 }`, so tools won't have to deal with `\t`. (As you see, this can have various results depending on the tool).

Reply:
ok, I just left as it was originally - I'll fix that now